### PR TITLE
fix: add linea mainnet to nft detection supported networks

### PR DESCRIPTION
**Pull request by sahar-fehri**
## Explanation

PR to add linea mainnet to the list of supported networks for nftDetection.

### `@metamask/assets-controllers`

- **ADDED**: Added linea mainnet chainId to the list of supported networks for nft auto-detection.

## Checklist

- [ ] I've updated the test suite for new or updated code as appropriate
- [ ] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [ ] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

**Review by mikesposito — APPROVED** [review]
Looks good!

File: packages/assets-controllers/src/NftDetectionController.ts [hunk 1]
```diff
@@ -56,7 +56,10 @@ export type NftDetectionControllerMessenger = RestrictedControllerMessenger<
   AllowedActions['type'],
   AllowedEvents['type']
 >;
-const supportedNftDetectionNetworks: Hex[] = [ChainId.mainnet];
+const supportedNftDetectionNetworks: Hex[] = [
+  ChainId.mainnet,
```
Comment (cryptodev-2s) [inline packages/assets-controllers/src/NftDetectionController.ts:60]: Do we have some tests that checks that we `detectNfts` only for theses two networks ? if not what do you think about adding some ?

**Review by cryptodev-2s — CHANGES_REQUESTED** [review]

File: packages/assets-controllers/src/NftDetectionController.ts (hunk 1 above)
Comment (sahar-fehri) [reply packages/assets-controllers/src/NftDetectionController.ts:60]: Thanks @cryptodev-2s!  i have added a unit test covering detecting NFTs on linea mainnet 🙏

**Review by cryptodev-2s — APPROVED** [review]
LGTM!

**Pull request merged**